policyengine_us/parameters/gov/states/de/tax/income/subtractions/pension_exclusion/cap/military.yaml

```diff
@@ -0,0 +1,15 @@
+description: Delaware allows an exclusion of taxable pension income for military pension income capped at this amount.
+values:
+    2021-01-01: 12_500
+metadata:
+  unit: currency-USD
+  period: year
+  label: Delaware Max Pension Exclusion Amount for military
+  reference:
+    - title: Delaware individual income tax instructions for 2021 Page 6 Line 6
```

Comment:
add a comment above the 2021 source saying:
# military retirement exclusion has been added in 2022